pkg/spoke/spokeagent.go

```diff
@@ -172,6 +173,7 @@ func (o *SpokeAgentOptions) RunSpokeAgent(ctx context.Context, controllerContext
 
 		go bootstrapInformerFactory.Start(bootstrapCtx.Done())
 		go spokeKubeInformerFactory.Start(bootstrapCtx.Done())
+		go namespacedSpokeKubeInformerFactory.Start(bootstrapCtx.Done())
```

Comment:
i think we need to restart the  namespacedSpokeKubeInformerFactory at line-278,  And we do not need to start spokeKubeInformerFactory during bootstrap. The spokeKubeInformerFactory will be started after bootstrap to retrieve nodes. So line-175 can also be removed.

Reply:
thanks for the clue, updated